source/content/wordpress-cron.md

```diff
@@ -71,62 +86,79 @@ terminus wp <site>.<env> -- cron <your wp-cron command and switches here>
 
 1. Set up an account
 1. Set up a job that calls `https://yourdomain.tld/wp-cron.php?doing_wp_cron`
 
 <Alert title="Note" type="info">
+
 Replace `yourdomain.tld` with your domain.
+
 </Alert>
 
-Depending on what service you use, you may have the ability to set up multiple jobs at different times. Creating a single job that calls your site's `wp-cron.php` script every 15 minutes is all you should need. WP-Cron will take care of the rest. If you create new schedules that need to be run more often than once every 15 minutes, you will need to adjust your job accordingly.
+You might have the ability to set up multiple jobs at different times depending on the service you use. Creating a single job that calls your site's `wp-cron.php` script every 15 minutes is all you should need. WP-Cron will take care of the rest. You will need to adjust your job if you create new schedules that need to run more frequently than every 15 minutes.
 
-It's important that you do not add a value to the `doing_wp_cron` query variable; it must be empty for the cron to work properly.
+<Alert title="Note" type="info">
+
+Do not add a value to the `doing_wp_cron` query variable. This variable must be empty for Cron to work correctly.
+
+</Alert>
 
 ### Using Your Own Server
 
-If you administer your own server, you can use the cron service to make a call to the `wp-cron.php` script. You will have to learn how to properly set up a Cron job and use something like `wget` or `curl` to fetch a web page. Unless you take special precautions, it is not any safer to use your own server vs. a web-based cron service; however, it does give you more control.
+ You can use the Cron service to make a call to the `wp-cron.php` script if you administer your own server. You will have to learn how to correctly set up a Cron job and use something like `wget` or `curl` to fetch a web page. Using your own server is not safer than using a web-based Cron service.
+ 
+## Troubleshooting 
 
-### Security
+### Problems With Low Traffic Sites
 
-For the most part, `wp-cron.php` is secure. There is only one parameter you can pass in that will affect the script (`doing_wp_cron`). This `$_GET` value is not filtered; it is only used as a flag and not as input for a process or variable. Beyond that, all input is ignored.
+Low traffic WordPress sites may experience skipped jobs because WP-Cron can't execute jobs if people are not visiting your site. This doesn't mean your page will be slow from previous jobs when someone eventually does visit your site. Regardless of how many jobs WP-Cron has to execute, all jobs are run in the background so your site's performance is not adversely affected.
+
+Low traffic WordPress sites using Pantheon Cron may also experience skipped jobs. For example, a site that receives no traffic for twelve hours might be put to sleep, which will stop Pantheon Cron jobs. Pantheon Cron will continue to run every hour regardless of traffic for twelve hours.
```

Comment:
In some cases, the timing will be one hour.

Per https://pantheon.io/docs/application-containers#idle-containers
> Pantheon containers spin down (sleep) after ~1 hour of idle time. Live environments on a paid plan will spin down after 12 hours of idle time.